.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "gce/serviceaccounts/samples2"]
 	path = gce/serviceaccounts/samples2
 	url = https://github.com/GoogleCloudPlatform/deploymentmanager-samples.git
+[submodule "charts/elasticsearch/dependencies/helm-charts"]
```

Comment:
Other remote chart usages in this repo so far would not include these as submodules, instead refering to the remote chart repository.
So it might be worth switching to that pattern here too https://github.com/wbstack/deploy/blob/main/k8s/helm/ingress-nginx/helmfile.yaml#L3